app/assets/javascripts/lib/vcalendar.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Create an vCalendar file with all the current courses on it,
+ * set to weekly, starting with the current week.
+ */
+var createVCalendar = function(periods) {
```

Comment:
Can you scope this in the YACS namespace? (maybe create sub-namespace e.g. `Yacs.util`)